src/index.tsx

```diff
@@ -32,3 +36,6 @@ interface TableInspectorProps extends ComponentProps<typeof TableInspector> {
 interface ObjectInspectorProps extends ComponentProps<typeof ObjectInspector> {
   table: false;
 }
+interface DomInspectorProps extends ComponentProps<typeof DOMInspector> {
+  table: false;
+}
```

Comment:
What's this doing?

Reply:
I think this is so that `<Inspector>` can accept `<DOMInspector>` props